content/redirects/cloudgov-vendors-2019.html

```diff
@@ -0,0 +1,5 @@
+---
+permalink: /assets/documents/cloudgov-vendors-2019.pdf/index.html
```

Comment:
`https://cloud.gov/assets/documents/cloudgov-vendors-2019.pdf` doesnt' exist.... is this the right source URL?

Reply:
ah, the permalink should say:

```
permalink: /resources/cloudgov-vendors-2019.pdf
```

Reply:
Updated the permalink. Good catch!  Where should the redirect go?